GCSViews/ConfigurationView/ConfigHWCompass2.cs

```diff
@@ -121,15 +124,15 @@ public void Activate()
             mavlinkComboBoxfitness.setup(ParameterMetaDataRepository.GetParameterOptionsInt("COMPASS_CAL_FIT",
                 MainV2.comPort.MAV.cs.firmware.ToString()), "COMPASS_CAL_FIT", MainV2.comPort.MAV.param);
 
-            mavlinkCheckBoxUseCompass1.setup(1, 0, "COMPASS_USE", MainV2.comPort.MAV.param);
-            mavlinkCheckBoxUseCompass2.setup(1, 0, "COMPASS_USE2", MainV2.comPort.MAV.param);
-            mavlinkCheckBoxUseCompass3.setup(1, 0, "COMPASS_USE3", MainV2.comPort.MAV.param);
+            mavlinkCheckBoxUseCompass1.setup(1, 0, get_compass_param_name("USE", 1), MainV2.comPort.MAV.param);

```

Comment:
there is 2 options here, use what you have done, or pass in both text versions, as the param name field can take an array (havn't checked this, but could be added, as param renames are common these days)